lib/ts/recipe/session/sessionClass.ts

```diff
@@ -162,4 +204,32 @@ export default class Session implements SessionContainerInterface {
             throw err;
         }
     };
+
+    getSessionGrants() {
+        return this.grants;
+    }
+    shouldRefetchGrant(grant: Grant<any>, userContext?: any): Awaitable<boolean> {
+        return grant.shouldRefetchGrant(this.getSessionGrants(), userContext);
+    }
+    async fetchGrant(grant: Grant<any>, userContext?: any): Promise<void> {
+        const value = await grant.fetchGrant(this.getUserId(), userContext);
+        if (value !== undefined) {
+            const newGrantPayload = grant.addToGrantPayload(this.getSessionGrants(), value, userContext);
+
+            await this.updateSessionGrants(newGrantPayload, userContext);
+        }
+    }
+    checkGrantInToken(grant: Grant<any>, userContext?: any): Awaitable<boolean> {
+        return grant.isGrantValid(this.getSessionGrants(), userContext);
+    }
+    async addGrant<T>(grant: Grant<T>, value: T, userContext?: any): Promise<void> {
+        const newGrantPayload = grant.addToGrantPayload(this.getSessionGrants(), value, userContext);
+
+        await this.updateSessionGrants(newGrantPayload, userContext);
+    }
+    async removeGrant<T>(grant: Grant<T>, userContext?: any): Promise<void> {
+        const newGrantPayload = grant.removeFromGrantPayload(this.getSessionGrants(), userContext);
+
+        await this.updateSessionGrants(newGrantPayload, userContext);
+    }
```

Comment:
I think the add, remove and update methods should be exposed here since it's the most convenient/simplest interface to add a claim to the current session.
```ts
session.addClaim(SecondFactorOTPClaim, true)
```